activerecord/lib/active_record/errors.rb

```diff
@@ -273,12 +273,13 @@ class StrictLoadingViolationError < ActiveRecordError
   #     def create
   #       Book.transaction do
   #         book = Book.new(params[:book])
+  #         # If this fails, user gets generic 500 behavior
```

Comment:
```suggestion
  #         # If this fails, the user gets generic 500 behavior.
```

What do you think about making the 500 more explicit?
For beginners who don't know 500 is a HTTP code.
```suggestion
  #         # If this fails, the user gets the 500 server error behavior.
```
or maybe:
```suggestion
  #         # If this fails, the user sees a "500 Server Error" message.
```

Reply:
perhaps problem with this is, user won't see that copy, because app is likely configured to show something nicer